Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:10.15.1-alpine AS builder
+
+COPY . /git-history
+
+WORKDIR /git-history
+
+RUN npm install -g yarn \
```

Comment:
Just curious, is this to upgrade yarn? It should already be in the node alpine images.

Reply:
@azz Sorry, I forgot that the yarn has been installed in the nodejs alpine image.

Reply:
@azz I have modified the code again.
we still need to confirm if git-history can complete OAuth2 authorization without relying on netlify